apps/choleskey/choleskey_stdpar.cpp

```diff
@@ -0,0 +1,78 @@
+// Cholesky Decomposition: stdpar
+#include "argparse/argparse.hpp"
+#include "commons.hpp"
+
+#include <algorithm>
+#include <execution>  // Include the execution header
+#include <experimental/mdspan>
+#include <iostream>
+#include <numeric>
+#include <vector>
+
+using namespace std;
+
+struct Multiply {
+  int operator()(int a, int b) const { return a * b; }
+};
+
+template <typename T>
+void Cholesky_Decomposition(std::vector<T>& vec, int n) {
+  std::vector<std::vector<T>> lower(n, std::vector<T>(n, 0));
+
+  using view_2d = std::extents<T, std::dynamic_extent, std::dynamic_extent>;
+  auto matrix = std::mdspan<T, view_2d, std::layout_right>(vec.data(), n, n);
+
+  Multiply multiplier;
+  // Decomposing a matrix into Lower Triangular
+  for (int i = 0; i < matrix.extent(0); i++) {
+    for (int j = 0; j <= i; j++) {
+      int sum = 0;
+
+      if (j == i)  // summation for diagonals
+      {
+
+        // apply transform_reduce for the first [lower[j].cbegin(), lower[j].cbegin() +j )
+        // sum = std::transform_reduce(std::execution::par, lower[j].cbegin(), lower[j].cbegin() +j, 0, std::plus{}, Square());
```

Comment:
nit: remove comments in the finalized PR.

Reply:
yes!